autopilot/financial-services/bankruptcy.ipynb

```diff
@@ -0,0 +1,728 @@
+    "            \"CandidateDefinitionNotebookLocation\"\n",
+    "        ]\n",
+    "    )\n",
+    "    print(f\"download DataExplorationNotebookLocation for {eval_obj}\" )\n",
+    "    print(\n",
+    "        automl.describe_auto_ml_job()[\"AutoMLJobArtifacts\"][\n",
+    "            \"DataExplorationNotebookLocation\"\n",
+    "        ]\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Wrapper function run_automl_process is called with objective AUC and F1. This wrapper function calls multiple functions to creare AutoML object, run training process, create model from best trained job and finally return predicted data**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def run_automl_process(eval_obj):\n",
+    "    timestamp_suffix = strftime(\"%d-%H-%M-%S\", gmtime())\n",
+    "    base_job_name = f\"{auto_ml_job_name}-{eval_obj}{timestamp_suffix}\"\n",
+    "    print(base_job_name)\n",
+    "    automl = create_automl_object(eval_obj, base_job_name)\n",
+    "    automl_fit(automl, base_job_name)\n",
+    "    check_status(automl)\n",
+    "    best_candidate_name, best_candidate = get_best_candidate(automl)\n",
+    "    model = create_model(automl, best_candidate_name, best_candidate, timestamp_suffix)\n",
+    "    transformer = create_transformer(model, best_candidate, eval_obj)\n",
+    "    pred_df = return_pred_df(transformer)\n",
+    "    pred_df.to_csv(f'data_{eval_obj}_bankruptcy.csv', index=False)\n",
+    "    download_notebooks(automl, eval_obj)\n",
+    "    return pred_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we are ready to run for the auto pilot job. We call the wrapper function run_automl_process with objective AUC and F1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"*********running with eval objective AUC***********\")\n",
+    "data_auc = run_automl_process(\"AUC\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"*********running with eval objective F1***********\")\n",
+    "data_f1 = run_automl_process(\"F1\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once the model is created, we run a Transform job to get inference (i.e Prediction about the default) from the test data set and save to S3. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we plot ROC - the Area under the Curve (AUC) for true positive (in this data set Bankrupt) vs false positive (predicted as Bankrupt but not Bankrupt in the ground truth). The higher the prediction quality of the classification model, the more the AUC curve is skewed to the top left."
```

Comment:
Just in case you reuse this phrase... it needs to be just so to get past some filters.

```suggestion
    "Now, we plot AUC-ROC curve - the Area under the Curve (AUC) for true positive (in this dataset Bankrupt) vs false positive (predicted as Bankrupt but not Bankrupt in the ground truth). The higher the prediction quality of the classification model, the more the AUC curve is skewed to the top left."
```